docs/_book.yaml

```diff
@@ -186,6 +186,8 @@ upper_tabs:
         path: /cirq/tutorials/google/echoes
       - title: "Visualizing Calibration Metrics"
         path: /cirq/tutorials/google/visualizing_calibration_metrics
+      - title: "Advanced calibration"
```

Comment:
nit: this doesn't really match the title given in the notebook

Reply:
updated to match notebook title